### ISSUE TITLE
GetUserSignInDataGraph.PS1 - Missing created date for onprem users skips those users

### ISSUE THREAD

**Issue by massiveloop** [12Knocksinna/Office365itpros#75]
_TLDR; Modify script to still pull in hybrid users with null/blank CreatedDate_

GetUserSignInDataGraph.PS1 script worked wonderfully for our needs of evaluating for stale guest users in our Azure AD!
I just wanted to raise something I came across that might be a useful enhancement is possible. We are hybrid for internal users (syncing with AAD Connect) and I noticed that all onprem users that were created before the existence of our tenant are excluded from the output. I'm assuming this is because of a dependency for the CreatedDate in the flow but wondering if this is something that can be modified so we can still get last signons. Not the end of the world though since I can merge the data across onprem/cloud if needed.
